hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -623,8 +617,8 @@ async def _execute_cancel(self, trading_pair: str, order_id: str) -> str:
             }
             await self._api_request(
                 method="delete",
-                path_url="cash/order",
-                params=api_params,
+                path_url=CONSTANTS.ORDER_PATH_URL,
+                data=api_params,
```

Comment:
Do verify if using the body parameters is correct for this particular endpoint.

Reply:
Confirmed.